src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.FruitServiceImpl;
+import core.basesyntax.service.Parser;
+import core.basesyntax.service.ParserImpl;
+import core.basesyntax.service.Reader;
+import core.basesyntax.service.ReaderImpl;
+import core.basesyntax.service.Writer;
+import core.basesyntax.service.WriterImpl;
+import core.basesyntax.strategy.AddOperationHandler;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        Map<String, OperationHandler> operationStrategyMap = new HashMap<>();
```

Comment:
Create enum Operation with fields BALANCE("b"), SUPPLY("s")
```suggestion
        Map<Operation, OperationHandler> operationStrategyMap = new HashMap<>();
```